assets/models/system/codellama-13b-instruct-hf/spec.yaml

```diff
@@ -3,15 +3,15 @@ name: CodeLlama-13b-Instruct-hf
 path: ./
 properties:
   datasets: Publicly available sources
-  finetuning-tasks: text-classification
-  inference-recommended-sku: Standard_NC24s_v3, Standard_ND40rs_v2, Standard_ND96asr_v4,
-    Standard_ND96amsr_A100_v4
+  inference-recommended-sku: Standard_NC24s_v3
+  evaluation-recommended-sku: Standard_NC24s_v3
```

Comment:
add all the computes supported for scenario as comma separated string

Reply:
Done